commit 7b2e2589c00c02ada1eb2e7289679da83954cec3
Author: Daniel Martinez <danielmartinez@cock.li>
Date:   Tue Feb 27 10:57:15 2024 -0500

    use posix basename to fix build on musl
    
    glibc provides a nonstandard basename implementation,
    this can be overriden and posix basename can be used
    by includeing libgen.h, however musl only has posix
    basename, and must always include libgen.h
    
    In this particular case, it doesn't appear that using
    the posix version of basename will cause any issues,
    as it is simply being used to match a hardcoded config
    file name.

diff --git a/daemon/gamemode-config.c b/daemon/gamemode-config.c
index 92f5406..a63459c 100644
--- a/daemon/gamemode-config.c
+++ b/daemon/gamemode-config.c
@@ -41,6 +41,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <ini.h>
 
 #include <dirent.h>
+#include <libgen.h>
 #include <math.h>
 #include <pthread.h>
 #include <pwd.h>
